hail/python/test/hail/matrixtable/test_matrix_table_from_parts.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+import hail as hl
+
+class Inits():
+    globals={'hello': 'world'}
+    rows={'foo': ['a', 'b']}
+    cols={'bar': ['c', 'd']}
+    entries={'baz': [[1, 2], [3, 4]]}
```

Comment:
These seem like they should be file-level variables. Am I misunderstanding the value of making them class-level variables?

Reply:
I'm trying to group test data within some namespace though doesn't seem to have worked well. I'll just make them file level bindings

Reply:
On reflection, I prefer the grouping. I think making the functions static methods also helps group together logical units of code.